### PR TITLE
Developer console

### PR DESCRIPTION
**Pull request by vertonghenb**

### CODE REVIEW

**Review by Paultje52 — APPROVED** [review]
Everything is translated correctly

**Comment by RMo-Sloth** [comment]
Some small modifications should be made to this translation. 

1. The translation tool used is mentioned below.
2. Some parts are translated too literal from English to sentences that have no meaning in Dutch. E.g. "De meeste ontwikkelaars leunen voor de"  on line 9 is improper Dutch.

File: 1-js/01-getting-started/4-devtools/article.md [hunk 1]
```diff
@@ -1,63 +1,67 @@
-# Developer console
+# Ontwikkelaarsconsole
 
-Code is prone to errors. You will quite likely make errors... Oh, what am I talking about? You are *absolutely* going to make errors, at least if you're a human, not a [robot](https://en.wikipedia.org/wiki/Bender_(Futurama)).
+De code is gevoelig voor fouten. Je zult waarschijnlijk fouten maken... Oh, waar heb ik het over? Je gaat *absoluut* fouten maken, tenminste als je een mens bent, niet een [robot](https://en.wikipedia.org/wiki/Bender_(Futurama)).
```
Comment (noah-vdv) [inline 1-js/01-getting-started/4-devtools/article.md:3]:
```suggestion
De code is gevoelig voor fouten. Je zult waarschijnlijk fouten maken... Oh, waar heb ik het over? Je gaat *absoluut* fouten maken, tenminste als je een mens bent, niet een [robot](https://nl.wikipedia.org/wiki/Bender_(Futurama)).
```

File: 1-js/01-getting-started/4-devtools/article.md [hunk 2]
```diff
@@ -1,63 +1,67 @@
...
 
-- Developer tools allow us to see errors, run commands, examine variables, and much more.
-- They can be opened with `key:F12` for most browsers on Windows. Chrome for Mac needs `key:Cmd+Opt+J`, Safari: `key:Cmd+Opt+C` (need to enable first).
+- Ontwikkelaarstools stellen ons in staat om fouten te zien, commando's uit te voeren, variabelen te onderzoeken en nog veel meer.
+- Ze kunnen worden geopend met `key:F12` voor de meeste browsers op Windows. Chrome voor Mac heeft `key:Cmd+Opt+J` nodig, Safari: `key:Cmd+Opt+C` (moet eerst worden ingeschakeld).
+
+Nu hebben we de omgeving klaar. In de volgende sectie gaan we naar JavaScript.
+
+*** Vertaald met www.DeepL.com/Translator (gratis versie)
+ ***
```
Comment (noah-vdv) [inline 1-js/01-getting-started/4-devtools/article.md:67]:
```suggestion
```

**Review by strangernr7 — CHANGES_REQUESTED** [review]
Wikipedia link to Dutch and removing non important info

**Pull request closed without merging**